### DISCUSSION TITLE
#100DaysOfGatsby-2021

### DISCUSSION THREAD

**Discussion by Officer-R** [gatsbyjs/gatsby#28807] [Help]
Hello guys, first of all Happy New year.

I just received a mail from @hashimwarren  regarding #100DaysofGatsby and I am super excited to learn a new skill. Hope it's  fun learning GatsbyJS.

Can anyone help me with the UI/UX design for the AudioC0RE website ? I am researching about the design, which headless CMS to use and many other things.

Can I use [Sanity](https://www.sanity.io) or am I **restricted** to use [contentful](https://www.contentful.com) as mentioned in the BRIEFING ?

Good to be here and learn with fellow developers.

Regards
Rohit

**Comment by hashimwarren** [comment]
Thanks for the question, Rohit.

You can choose any CMS! We gave a specific recommendation because we found last year that people got hung up in decision fatigue trying to pick one.

Sanity is probably a string choice!

As for the design, we're going to tackle that in an upcoming challenge. "Senior Dev" has a few requirements for performance, accessibility, and SEO that you'll need to keep in mind with the design 😉

**Reply by Officer-R** [reply]
Thank you for the quick response @hashimwarren
